aimpoint_mon/plot_aimpoint.py

```diff
@@ -165,12 +165,6 @@ def det_title(self):
     def ccd(self):
         return 'I3' if self.det.lower().endswith('i') else 'S3'
 
-    @property
-    def argsort(self):
-        if not hasattr(self, '_argsort'):
-            self._argsort = self.grouped.groups.aggregate(np.argsort)
```

Comment:
Aggregation that produces a ragged array result (lists w/ different lengths) instead of a scalar was always a bit dodgy. Numpy will stop supporting array creation like that. The new code is simpler and more direct.